lib/applicationinsights.json

```diff
@@ -0,0 +1,6 @@
+{
+  "connectionString": "${file:/mnt/secrets/am/app-insights-connection-string}",
+  "role": {
+    "name": "HMCTS Access Management Judicial Booking Service"
```

Comment:
NB: The other PRs have gone with the short name for the service: e.g. 
* am-role-assignment-service
* am-org-role-mapping-service
* am-role-assignment-refresh-batch